engine/src/main/java/org/terasology/engine/world/chunks/localChunkProvider/LocalChunkProvider.java

```diff
@@ -161,14 +160,12 @@ public void setWorldEntity(EntityRef worldEntity) {
 
 
     private void processReadyChunk(final Chunk chunk) {
-        if (chunkCache.get(chunk.getPosition(new Vector3i())) != null) {
-            return; // TODO move it in pipeline;
-        }
```

Comment:
Chunk does not allow for getPosition to be null and will error out if attempted.